operators/deploy/tenant-operator/k8s-cluster-role.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: crownlabs-tenant-operator
+rules:
+  - apiGroups: ["crownlabs.polito.it"]
+    resources: ["workspaces", "workspaces/status"]
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+
+  - apiGroups: [""]
+    resources: ["namespaces"]
+    verbs: ["get", "list", "watch"]
+
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get", "list", "watch", "create"]
```

Comment:
Are these permissions required?

Reply:
I believe they are cause I placed the username and psw for the keycloak user in a secret (in the template)

Reply:
Ok, in that case you do not need these permissions (as the values are read directly by kubernetes when the container is started, and not by you inside the operator)

Reply:
You are right 🤦🏻‍♂️  thanks